terracumber_config/tf_files/Uyuni-Master-NUE.tf

```diff
@@ -117,6 +117,7 @@ module "cucumber_testsuite" {
       provider_settings = {
         mac = "aa:b2:93:01:00:d1"
       }
+      additional_repos = { java11 = "http://download.suse.de/ibs/home:/fstrba:/branches:/SUSE:/SLE-15:/Update/SUSE_SLE-15_Update/" }
```

Comment:
FMPOV: @ktsamis also asking for an openSUSE 15.2 target to get a build this OS.

Yeah, I know. SLE15SP2 should be the same. But should is not the same as `is`. He can add a repo with a path to `openSUSE.org:openSUSE:Leap:15.2:Update`.

Reply:
This one should work right? https://download.opensuse.org/repositories/Java:/Factory/openSUSE_Leap_15.2/

Reply:
For each SP the Devel of Java/Factory has the update already, the only one that comes from the home project is 15sp3 which is not released.

Reply:
For now we don't add it to uyuni that will come tomorrow after releasing the latest uyuni release today